internal/format/base_visitor.go

```diff
@@ -73,7 +73,37 @@ func (v *baseVisitor) PComment(comment *proto.Comment) {
 	}
 }
 
-func (v *baseVisitor) POptions(isFieldOption bool, options ...*proto.Option) {
+func (v *baseVisitor) POptions(options ...*proto.Option) {
+	v.pOptions(false, options...)
+}
+
+// fieldType can be "" in case of enum field
+func (v *baseVisitor) pMessageOrEnumField(prefix string, fieldName string, fieldType string, fieldTag int, comment *proto.Comment, inlineComment *proto.Comment, options ...*proto.Option) {
+	if fieldType != "" {
+		fieldType = fieldType + " "
+	}
+	v.PComment(comment)
+	if len(options) == 0 {
+		v.PWithInlineComment(inlineComment, prefix, fieldType, fieldName, " = ", fieldTag, ";")
+		return
+	}
+	if len(options) == 1 {
+		o := options[0]
+		if len(o.Constant.Array) == 0 && len(o.Constant.OrderedMap) == 0 {
```

Comment:
Seeing as this is used in `pOptions` as well and has an issue associated with it, we might want to make a more explanatory helper function. I'm not sure what a good name for this would be:

```
func <someName>(o *proto.Option) bool {
  return len(o.Constant.Array) == 0 && len(o.Constant.OrderedMap) == 0 && o.Constant.SourceRepresentation() != ""
}
```

Reply:
Done